moncli/types.py

```diff
@@ -125,6 +127,45 @@ class TextType(MondayType):
     null_value = ""
 
 
+class DateType(MondayType):
+
+    native_type = datetime
+    allow_casts = (str, int)
+    null_value = {}
+    has_time = False
+
+    def __init__(self, id: str = None, title: str = None, has_time: bool = False, *args, **kwargs):
+        self.id = id
+        self.title = title
+        self.has_time = has_time
+        super().__init__(self.id, self.title, *args, **kwargs)
+
+    def _cast(self, value):
+        if isinstance(value, int):
+            try:
+                return datetime.fromtimestamp(value)
+            except ValueError:
+                raise ConversionError('Invalid UNIX timestamp "{}".'.format(value))
+        if isinstance(value, str):
+            try:
+                date = datetime.strptime(value.split(' ', 1)[0], DATE_FORMAT)
+            except ValueError:
+                raise ConversionError('Cannot convert value "{}" into Date.'.format(value))
+            if self.has_time == True:
+                try:
+                    time = datetime.strptime(value.split(' ', 1)[1], TIME_FORMAT)
+                    date = date + timedelta(hours=time.hour, minutes=time.minute, seconds=time.second)
+                    return date
+                except ValueError:
+                    raise ConversionError('Cannot convert value "{}" into Time.'.format(value))
+
+    def _export(self, value):
+        if self.has_time == True:
+            value = value.replace(tzinfo=timezone.utc)
```

Comment:
I thought that this nifty trick would work, but it does not actually convert the underlying values as hoped.  It simply swaps out the timezone.  To make the proper conversion, please use the following:

```python
value = value.astimezone(timezone.utc)
```